### PR TITLE
SONAR-12717 Add a helper to hide footer

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**
For the new Hotspots page that has no footer

**Checklist**

* [x] Functional validation
* [x] Update changelog

### CODE REVIEW

**Review by wouter-admiraal-sonarsource — APPROVED** [review]
LGTM, but _why_ are we removing the footer on specific pages? Especially on a single page? Did Matt intend that?

File: components/controls/BoxedTabs.tsx [hunk 1]
```diff
@@ -43,13 +48,11 @@ const StyledTab = styled.button<{ active: boolean }>`
...
   min-height: 56px;
-  cursor: pointer;
+  ${props => !props.active && 'cursor: pointer;'}
   outline: 0;
   padding: calc(2 * ${props => props.theme.sizes.gridSize});
 
-  &:hover {
-    background-color: ${props => props.theme.colors.barBackgroundColorHighlight};
-  }
+  ${props => (!props.active ? highlightHoverMixin(props.theme) : null)}
```
Comment (philippe-perrin-sonarsource) [inline components/controls/BoxedTabs.tsx]: I think you can avoid passing the `props.theme` and assume in the mixin that the parameters is `props: { theme: ThemedProps }`.

Comment (philippe-perrin-sonarsource) [inline components/controls/BoxedTabs.tsx]: NP: I think you can avoid passing the `props.theme` and assume in the mixin that the parameter is `props: { theme: ThemedProps }`.

**Review by philippe-perrin-sonarsource — APPROVED** [review]
LGTM. NP only.

**Comment by gregaubert** [comment]
@jeremy-davis-sonarsource Could you also share the related code on SQ side about the helper to hide a footer ? Because this ends up in sonar-ui-common but we could not use it in SC currently since the class is defined on SQ side only.

**Pull request merged**